lib/commonmarker/config.rb

```diff
@@ -7,23 +7,28 @@ module Config
     OPTS = {
       parse: {
         DEFAULT: 0,
+        SOURCEPOS: (1 << 1),
+        UNSAFE: (1 << 17)
         VALIDATE_UTF8: (1 << 9),
         SMART: (1 << 10),
         LIBERAL_HTML_TAG: (1 << 12),
         FOOTNOTES: (1 << 13),
         STRIKETHROUGH_DOUBLE_TILDE: (1 << 14),
-        UNSAFE: (1 << 17)
       }.freeze,
       render: {
         DEFAULT: 0,
         SOURCEPOS: (1 << 1),
         HARDBREAKS: (1 << 2),
+        UNSAFE: (1 << 17),
         NOBREAKS: (1 << 4),
+        VALIDATE_UTF8: (1 << 9),
```

Comment:
I copied over all of the `parse` options and resorted the `render` hash. As described in the PR description, since the render method also initializes the parser with the same options integer, all of the parse options should be available as render steps as well.